README.md

```diff
@@ -235,6 +235,15 @@ like this:
 You'll probably get a more reliable experience if you use
 [forever](https://npmjs.org/package/forever) to keep the daemon running.
 
+### Arguments
+
+Execute with the config file on the command line using -c
```

Comment:
The wording of this implies that you _have_ to pass the `-c` flag, which isn't true.

Perhaps something like:

> You may specify a configuration location other than the default with the `-c` flag:

The default locations don't really belong in the "Arguments" section.